### PR TITLE
Regression CI and fixup

### PR DESCRIPTION
**Pull request by aparcar**

### CODE REVIEW

**Comment by aparcar** [comment]
This now also verifies that 100 keys were indeed exchanged.

**Comment by aparcar** [comment]
@prabhpreet thanks again for your initial work on this!

File: .ci/run-regression.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```
Comment (emilengler) [inline .ci/run-regression.sh:1]: `#!/usr/bin/env bash` please for better compatibility with Nix.

File: .ci/run-regression.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+iterations="$1"
+sleep_time="$2"
+
+PWD="$(pwd)"
+EXEC="$PWD/target/release/rosenpass"
+LOGS="$PWD/output/logs"
```
Comment (emilengler) [inline .ci/run-regression.sh:8]: Maybe `mktemp(1)` is better than an `output/` directory? Just a suggestion, I do not have a strong opinion about either approach.

File: misc/generate_configs.py [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
-
+import os
 
 config = dict(
     peer_counts=[1, 5, 10, 50, 100, 500],
     peer_count_max=100,
-    ate_ip="192.168.2.1",
-    dut_ip="192.168.2.4",
+    ate_ip="127.0.0.1",
+    dut_ip="127.0.0.1",
```
Comment (emilengler) [inline misc/generate_configs.py:9]: Can we test IPv6 as well?

**Review by emilengler — CHANGES_REQUESTED** [review]

**Review by koraa — APPROVED** [review]

File: misc/generate_configs.py (hunk 3 above)
Comment (aparcar) [reply misc/generate_configs.py:9]: Sure but it wasn't part of the regression

File: .ci/run-regression.sh (hunk 2 above)
Comment (aparcar) [reply .ci/run-regression.sh:8]: Since it's running in the CI, it's a fresh folder anyway.

File: .ci/run-regression.sh (hunk 1 above)
Comment (aparcar) [reply .ci/run-regression.sh]: Fixed.

**Comment by aparcar** [comment]
@emilengler I think this is ready, please feel free to merge

**Comment by emilengler** [comment]
could you update the branch?

**Pull request merged**